src/sql/src/func.rs

```diff
@@ -1988,6 +1988,9 @@ lazy_static! {
                         column_names: vec![Some("unnest".into())],
                     })
                 }), oid::FUNC_UNNEST_LIST_OID;
+            },
+            "mz_sleep" => Scalar {
+                params!(Float64) => UnaryFunc::Sleep, 2626;
```

Comment:
Can you add this to the `MZ_INTERNAL_BUILTINS` list below instead? I think it would be best if we really discourage users from calling this.

Reply:
Also, looks like 2626 is the OID for `pg_sleep`, but I think we should allocate a new OID (as we do with e.g. `FUNC_MZ_ALL_OID`) since this function has different semantics and a different name from `pg_sleep`!